spec/models/question_spec.rb

```diff
@@ -50,4 +50,22 @@
       expect(subject.response_choices).to include(subject.question_orders.first.response_choice)
     end
   end
+
+  context 'question renders as sms' do
+    subject { FactoryGirl.create(:question, :with_multiple_response_choices) }
+    expectedString = "hello world?\na) first choice\nb) second choice"
+    
+    it 'with response choices' do
+      expect(subject.format_as_sms).to eq(expectedString)
+    end
+  end
+
+  context 'question renders as sms' do
```

Comment:
Context is usually used to describe the environment you're going to set up for this test. In this case, a good context string (to differentiate from the other context you're creating) might be `context 'without response choices'` and the test string would indicate what you're actually checking in the test.